src/omd.mli

```diff
@@ -59,11 +59,14 @@ and block_desc =
   | Link_def of string link_def
   | Definition_list of def_elt list
 
-type doc = block list
+type doc = Ast.block list
```

Comment:
Why this change?

Reply:
Just needed to expose the fact that `block` is the same as `Ast.block` to the type checker, so that it could recognize it's occurrence in the reexported `Html` module.

However, since the plan is to just build the `Tyxml` off of the `Omd.doc`, this will be a mute point.